recipes/libucl/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import CMake, ConanFile, tools
+import functools
+
+required_conan_version = ">=1.33.0"
+
+
+class LibuclConan(ConanFile):
+    name = "libucl"
+    description = "Universal configuration library parser "
+    license = "BSD-2-Clause"
+    homepage = "https://github.com/libusb/libusb"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "libusb", "usb", "device")
```

Comment:
conan is not a proper topic, checked by hook KB-H064

Reply:
The topics are also completely wrong.
I used the libusb recipe as a template.
```suggestion
    topics = ("universal", "configuration", "language", "parser", "ucl")
```